pom.xml

```diff
@@ -436,21 +436,11 @@ Additionally it includes CPD, the copy-paste-detector. CPD finds duplicated code
                         <testSourceDirectory>${project.basedir}/src/test/kotlin</testSourceDirectory>
                     </configuration>
                     <dependencies>
-                        <!-- Junit 5 = Platform + Jupiter (5) + Vintage (3 & 4)-->
-
-                        <!-- Junit platform -->
-                        <!-- Needed to use kotlintest -->
-                        <dependency>
```

Comment:
I think that wasn't optional. The only tests run now are the kotlin tests:

https://travis-ci.org/pmd/pmd/jobs/501059616#L2135-L2146

Reply:
Hm... I got a deprecation warning, that the platform is now included in surefire, that why I removed it....

Reply:
This is solved, now all tests are executed again:
https://travis-ci.org/pmd/pmd/jobs/504377286#L3191

Surefire now provides a native provider for junit5.
However, in order to have junit4 tests being executed, one needs to add the junit-vintage-enginge on the test classpath of the project (apparently a transitive dependency doesn't work).
Surefire selects automatically the correct engine depending on what is available on the classpath. Since kotlintests brings in junit5, surefire now only executed junit5 tests...
At the moment "pmd-java" is the only module that uses "pmd-lang-test", where this is relevant.